listView.php

```diff
@@ -186,9 +195,10 @@
 // function to refresh the modal page for task details
 function dynamicModal(str)
 {
-$("#viewTaskFrame").attr("src", "https://mansci-db.uwaterloo.ca/~wmmeyer/wmmeyer/viewTask.php?id="+str);
+$("#viewTaskFrame").attr("src", "https://mansci-db.uwaterloo.ca/~wmmeyer/r2/studentplanner/viewTask.php?id="+str);
```

Comment:
Confirm the URL with other members - these were reverted in https://github.com/MayyarAA/studentplanner/pull/39/files